client/src/pages/donate-other.js

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable max-len */
+import React, { Component, Fragment } from 'react';
+import Helmet from 'react-helmet';
+import ReactGA from '../analytics/index.js';
```

Comment:
We have an import hierarchy, which is common across many projects.

1. Native (`fs`, `path`, `http`)
2. Module (`react`, `lodash`, `express`)
3. One line of separation
3. Local (anything created in the current project) 

This helps other contributors quickly find out if the module they are looking for is already imported or not. It might seem like nothing, but imagine one module with 15+ imports. The grouping really helps.